src/ParaTest/Coverage/CoverageReporter4.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace ParaTest\Coverage;
+
+use SebastianBergmann\CodeCoverage\CodeCoverage;
+use SebastianBergmann\CodeCoverage\Report\Clover;
+use SebastianBergmann\CodeCoverage\Report\Html\Facade;
+use SebastianBergmann\CodeCoverage\Report\PHP;
+
+class CoverageReporter4 implements CoverageReporterInterface
```

Comment:
minor... Maybe name this just `CoverageReporter` since it is the latest and greatest for the foreseeable future, but keep `CoverageReporter3` as is since it is "legacy"